gulpfile.js

```diff
@@ -94,20 +83,12 @@ gulp.task('sass', ['sass:format'], function () {
         .pipe(rename('Coveo.Slider.css'))
         .pipe(sourcemaps.write('../css'))
         .pipe(gulp.dest('./dist/css'))
-});
-
-gulp.task('sass:format', function () {
-    return gulp.src(['./scss/**/*.scss'])
-        .pipe(csscomb())
-        .pipe(gulp.dest('./scss'));
-});
-
+}));
 
 gulp.task('watch', function () {
     gulp.watch('./src/**/*.ts', ['ts']);
     gulp.watch('./test/src/**/*.ts', ['test:compile']);
     gulp.watch('./scss/**/*.scss', ['sass']);
 });
 
-gulp.task('default', ['ts', 'sass', 'test'], function () {
-});
+gulp.task('default', gulp.series('ts', 'sass', 'test'));
```

Comment:
just for fun, can we do something like 
`gulp.series(gulp.parallel('ts', 'test'), 'sass')`

Reply:
Yes!